Components/Open/ChatList/ChatItem/ChatItem.tsx

```diff
@@ -0,0 +1,84 @@
+'use client';
+
+import { Typography } from '@material-tailwind/react';
+import Image from 'next/image';
+import icon_people from '@/public/icon_people.svg';
+import { ChatItemProps } from '../ChatList.type';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import { koreanTime } from '@/utils/changeKoreanTime';
+import OpenChatPicture from '@/Components/Search/OpenChatPicture';
+
+const ChatItem = ({ chat }: ChatItemProps) => {
+	const pathName = usePathname();
+
+	const checkIsPrivate = () => {
+		if (pathName === '/private') {
+			return true;
+		}
+		return false;
+	};
```

Comment:
```suggestion
	const checkIsPrivate = () => pathName === '/private'
```

로만 간단하게 표현해도 괜찮을 것 같아요!

Reply:
감사합니다!! 저렇게 사용할 수 있다는 것을 전혀 모르고 있었습니다. 불필요한 코드 작성을 줄일 수 있을 것 같습니다. 대신 함수 명을 봐도 알 수 있게 `prefix`를 잘 활용하도록 하겠습니다.

Reply:
그쵸 함수일 필요도 없죠 사실! isPrivatePath 와 같이 처리해도 됩니다.